### ISSUE TITLE
manual field definition not working with New, edit

### ISSUE THREAD

**Issue by jackfruh** [Laravel-Backpack/CRUD#1644]
# Bug report

### What I did:
commented out `$this->crud->setFromDb();`
added columns with `addColumn`

````
 public function setup()
    {
        /*
        |--------------------------------------------------------------------------
        | CrudPanel Basic Information
        |--------------------------------------------------------------------------
        */
        $this->crud->setModel('App\Models\ResourceGroupRequest');
        $this->crud->setRoute(config('backpack.base.route_prefix') . '/resourcegroup');
        $this->crud->setEntityNameStrings('Resource Group Request', 'Resource Group Requests');

        /*
        |--------------------------------------------------------------------------
        | CrudPanel Configuration
        |--------------------------------------------------------------------------
        */

        // TODO: remove setFromDb() and manually define Fields and Columns
        // $this->crud->setFromDb();
        $this->crud->addColumn([
            'label' => 'Status',
            'type' => 'text',
            'name' => 'status',
            'priority' => 1,

        ]);
        $this->crud->addColumn('ResourceGroupName');
        $this->crud->addColumn('Subscription');
        $this->crud->addColumn('RequesterEmail');
        $this->crud->addColumn('created_at', 'Created');

        // add asterisk for fields that are required in ResourceGroupRequestRequest
        $this->crud->setRequiredFields(StoreRequest::class, 'create');
        $this->crud->setRequiredFields(UpdateRequest::class, 'edit');
        $this->crud->addFilter([
            'type' => 'simple',
            'name' => 'trashed',
            'label' => 'Show Deleted Records'
        ],
        false,
        function($values)
        { 
            // IF THE FILTER IS ACTIVE
            $this->crud->query->onlyTrashed();

        });
    }
````

### What I expected to happen:
Expect to see the 5 columns I specified on *all* Crud views (index, edit, create)

### What happened:
**Index** looks great, the table displays just as expected.
'Edit' and 'Add new record' both display no fields, just the submit/cancel buttons.

### What I've already tried to fix it:
Read documentation on $this->crud->addColumn 

**note** this works fine if I leave `$this->crud-setFromDB()` in the code.

### Backpack, Laravel, PHP, DB version:
latest

**Comment by tswonke** [comment]
For create and Update you have to definr fields: https://backpackforlaravel.com/docs/3.4/crud-fields

Columns are only for index 😉

**Comment by jackfruh** [comment]
Closing this as I just noticed in the documentation that addColumn and addField are actually two distinct things, so it seems it's working as expected.

@tabacitu if you happen to see this, it would be neat if there was a way to set fields and columns from the db independently.

ie 
````
//old:
$this->crud->setFromDb();

//new:
$this->crud->setIndexColumnsFromDb();
$this->crud->setFormFieldsFromDb();
````

This would accomplish the same thing, while making the scope of commenting these out clearer.
It also would have the added benefit of allowing a dev to easily create a custom index while relying on `setFormFieldsFromDb()` for the Create and Update screens.

Thanks!
